apps/frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./auth-layout.module.css";
+
+interface AuthLayoutProperties {
+	children: React.ReactNode;
+}
+
+const AuthLayout = ({ children }: AuthLayoutProperties) => {
```

Comment:
Let's use the same approach everywhere
```suggestion
const AuthLayout: React.FC<AuthLayoutProperties> = ({ children }) => {
```

Reply:
In that case, I’m getting the error: `'children' is missing in props validation`. How can I fix it?